SU2_CFD/src/numerics/turbulent/turb_sources.cpp

```diff
@@ -153,19 +153,19 @@ CNumerics::ResidualType<> CSourcePieceWise_TurbSA::ComputeResidual(const CConfig
 
 //    BC model constants
       chi_1 = 0.002;
-      chi_2 = 5.0;
+      chi_2 = 50.0;
 
       nu_t = (TurbVar_i[0]*fv1); //S-A variable
-      nu_cr = chi_2/rey;
-      nu_BC = (nu_t)/(vmag*dist_i);
+      //nu_cr = chi_2/rey;
+      //nu_BC = (nu_t)/(vmag*dist_i);
 
       re_v   = ((Density_i*pow(dist_i,2.))/(Laminar_Viscosity_i))*Omega;
       re_theta = re_v/2.193;
       re_theta_t = (803.73 * pow((tu + 0.6067),-1.027)); //MENTER correlation
       //re_theta_t = 163.0 + exp(6.91-tu); //ABU-GHANNAM & SHAW correlation
```

Comment:
Would it make sense to have an option for the different corrections?

Reply:
I do not think it would be necessary.

Reply:
Then the commented code should be deleted, comments have a tendency for obsolescence.
Also around line 5532 of CConfig.cpp there should me a message stating the model being used has the revised coefficients.